dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/Grid/BaseClasses/IEEE14Base.mo

```diff
@@ -0,0 +1,261 @@
+    Placement(visible = true, transformation(origin = {-70, -60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB2B3(BPu = 4.599875E-4 * ZBASE1, GPu = 0, RPu = 2.23719 / ZBASE1, XPu = 9.42535 / ZBASE1) annotation(
+    Placement(visible = true, transformation(origin = {10, -160}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB2B4(BPu = 3.57068E-4 * ZBASE1, GPu = 0, RPu = 2.76662 / ZBASE1, XPu = 8.3946 / ZBASE1) annotation(
+    Placement(visible = true, transformation(origin = {10, -120}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB2B5(BPu = 3.63369E-4 * ZBASE1, GPu = 0, RPu = 2.71139 / ZBASE1, XPu = 8.27843 / ZBASE1) annotation(
+    Placement(visible = true, transformation(origin = {-52, -100}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB3B4(BPu = 1.344255E-4 * ZBASE1, GPu = 0, RPu = 3.19035 / ZBASE1, XPu = 8.14274 / ZBASE1) annotation(
+    Placement(visible = true, transformation(origin = {100, -112}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Electrical.Lines.Line LineB4B5(BPu = 0, GPu = 0, RPu = 0.635593 / ZBASE1, XPu = 2.00486 / ZBASE1) annotation(
+    Placement(visible = true, transformation(origin = {28, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB6B11(BPu = 0, GPu = 0, RPu = 0.18088 / ZBASE2, XPu = 0.378785 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {-30, 90}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Electrical.Lines.Line LineB6B12(BPu = 0, GPu = 0, RPu = 0.23407 / ZBASE2, XPu = 0.487165 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {-70, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB6B13(BPu = 0, GPu = 0, RPu = 0.125976 / ZBASE2, XPu = 0.248086 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {-50, 120}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB7B8(BPu = 0, GPu = 0, RPu = 0, XPu = 0.33546 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {150, 20}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Electrical.Lines.Line LineB7B9(BPu = 0, GPu = 0, RPu = 0, XPu = 0.209503 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {110, 50}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Electrical.Lines.Line LineB9B10(BPu = 0, GPu = 0, RPu = 0.060579 / ZBASE2, XPu = 0.160922 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {60, 90}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Electrical.Lines.Line LineB9B14(BPu = 0, GPu = 0, RPu = 0.242068 / ZBASE2, XPu = 0.514912 / ZBASE2) annotation(
+    Placement(visible = true, transformation(origin = {50, 180}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  // Transformers
```

Comment:
For the transformers, are we sure it should be Zbase2 instead of ZBase1 ? Have you checked the side of the R,X in the model and the connections ?

Reply:
yes